src/Microsoft.DotNet.Build.Tasks.Packaging/src/GenerateRuntimeGraph.cs

```diff
@@ -194,6 +190,8 @@ public RuntimeGroup(ITaskItem item)
                 Architectures = item.GetStrings(nameof(Architectures));
                 AdditionalQualifiers = item.GetStrings(nameof(AdditionalQualifiers));
                 OmitRIDs = new HashSet<string>(item.GetStrings(nameof(OmitRIDs)));
+                OmitRIDs = new HashSet<string>(item.GetStrings(nameof(OmitRIDDefinitions)));
```

Comment:
I think this is supposed to write to `OmitRIDDefinitions` and `OmitRIDReferences`, not overwrite OmitRIDs each time.

Reply:
Doh, good catch